feconf.py

```diff
@@ -840,6 +856,8 @@ def get_empty_ratings():
 IMPROVEMENTS_URL_PREFIX = '/improvements'
 IMPROVEMENTS_HISTORY_URL_PREFIX = '/improvements/history'
 IMPROVEMENTS_CONFIG_URL_PREFIX = '/improvements/config'
+INCOMING_ANDROID_FEEDBACK_REPORT_URL = (
+    '/app_feedback_report/incoming_android_report')
```

Comment:
/appfeedbackreporthandler/... (follow existing conventions).

Reply:
Done, also updated on the Android side too (https://github.com/oppia/oppia-android/pull/2878)